### PR TITLE
TEMPORARY file to trigger CI tests

### PR DESCRIPTION
**Pull request by crlishka**
Tests for CI

PLEASE NOTE that I had to remove the WORK-IN-PROGRESS in the title because I want to force CI testing.

### CODE REVIEW

**Comment by crlishka** [comment]
TESTNOW

**Comment by crlishka** [comment]
TESTNOW

**Comment by crlishka** [comment]
TESTNOW

File: test/ci/README-ci-test-trigger.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This is a file being used to trigger tests of the CI system.  It will never be
+committed to master, and will be removed when the testing is done.
```
Comment (sayantan-nervana) [inline test/ci/README-ci-test-trigger.txt]: This file feels sad :'( and sees no purpose in life now, because of Chris's absolute refusal to commit it 
:-P

**Comment by crlishka** [comment]
This temporary test PR is no longer needed.  Closing.

**Pull request closed without merging**